go.mod

```diff
@@ -333,7 +334,7 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.45.0-rc1.0.20230908065430-05346fa12992
```

Comment:
Keeping as draft until this is replacable after having https://github.com/osmosis-labs/cosmos-sdk/pull/489 merged in